src/config/navigation/config.ts

```diff
@@ -19,20 +26,14 @@ if (globalSettings.navigation != 'drawer') {
     config: tabNavigatorConfig
   };
 
-  linkingConfig = {
-    prefixes: [Linking.createURL('/')],
-    config: {
-      screens: {
-        // For tab navigation choose the preferred tab by its position in the config array
-        [`Stack${index}`]: {
-          // The initialRouteName has to be the initial route of the chosen stack
-          // (Home for drawer navigation, and whatever is specified in the tab config for tab navigation)
-          initialRouteName: navigatorConfig.config.tabConfigs[index].stackConfig.initialRouteName,
-          screens: {
-            [ScreenName.EncounterUserDetail]: 'encounter',
-            [ScreenName.Home]: '*'
-          }
-        }
+  linkingConfig.config = {
```

Comment:
this are cool changes, just dont get what `.config` does/where it comes from in this part, same below so that the code works like it does now

Reply:
`linkingConfig` is a json object with entries. one entry has the key `config`.

```
linkingConfig = {
  config: "xyz"
}
```

defines the same as

```
linkingConfig.config = "xyz"
```

Reply:
okay, thank you for the explanation 😄